test/node/connection-handshake.js

```diff
@@ -126,3 +126,17 @@ test.test('must not accept handshakes on a client', (test) => {
   const connection = new jstp.Connection(transport, null, {});
   transport.emitPacket(handshake);
 });
+
+test.test(
```

Comment:
There is no check that the connection was dropped after `HANDSHAKE_TIMEOUT` and exactly because of the timeout and not some kind of error. I think it's better to use 'handshakeTimeout' server event and check the `id` of a dropped connection. We may also add time checking (that it happened after time >`HANDSHAKE_TIMEOUT`) but that may not be necessary in my opinion.